src/unistd.rs

```diff
@@ -612,6 +612,24 @@ pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<Uid>, group: Option<Gi
     Errno::result(res).map(drop)
 }
 
+/// Change the ownership of the file descriptor `fd` to be owned by the specified
+/// `owner` (user) and `group` (see
+/// [fchown(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/chown.html)).
+///
+/// The owner/group for the provided file descriptor will not be modified if `None` is
+/// provided for that argument.  Ownership change will be attempted for the path
+/// only if `Some` owner/group is provided.
+///
+/// The fchown() call cannot be used to portably change the ownership
```

Comment:
What exactly is the problem with unix domain sockets?  Do you have a reference?

Reply:
I don't. It's in the original request for this feature. I'll look it up...

Reply:
https://lkml.org/lkml/2004/11/1/84

Reply:
On the BSDs, OSX, and Illumos that returns `EINVAL`.  Only on Linux does it silently fail.  In any case I think this detail is too obscure to warrant mentioning in Nix's documentation.

Reply:
Comment is removed